ci/playbooks/customize_whitebox_image.yml

```diff
@@ -0,0 +1,36 @@
+---
+- hosts: controller, primary
+  name: Download neutron whitebox tempest plugin image and virt customize it
+  tasks:
+    - name: Install libguestfs package
+      become: true
+      ansible.builtin.package:
+        name:
+          - libguestfs
+          - libguestfs-tools
+          - libguestfs-tools-c
+        state: present
+
+    - name: Create whitebox directory
+      ansible.builtin.file:
+        path: "{{ ansible_user_dir }}/whitebox"
+        state: directory
+
+    - name: Download whitebox testing related image
+      ansible.builtin.get_url:
+        url: "https://dl.rockylinux.org/vault/rocky/9.3/images/x86_64/Rocky-9-GenericCloud.latest.x86_64.qcow2"
+        dest: "{{ ansible_user_dir }}/whitebox/Rocky-9-GenericCloud.latest.x86_64.qcow2"
+        mode: '0755'
+
+    - name: Install packages and set root password on the downloaded image
+      ansible.builtin.command:
+        cmd: >-
+          sudo virt-customize -a {{ ansible_user_dir }}/whitebox/Rocky-9-GenericCloud.latest.x86_64.qcow2
+          --install nmap,python3,keepalived,iperf3
```

Comment:
Hi @raukadah,

Trying to install 'python3' on rocky image caused issues recently, so it was removed from whitebox plugin devstack job [1].
rocky9.3 image should have 'python3' pre-installed [2], so it can be omitted.

[1] 922884: Remove install python3 when customizing image | https://review.opendev.org/c/x/whitebox-neutron-tempest-plugin/+/922884
[2] https://dl.rockylinux.org/vault/rocky/9.3/BaseOS/x86_64/os/Packages/p/python3-3.9.18-1.el9_3.1.x86_64.rpm

Reply:
Thank you for the comment. Test image is uploaded here: https://chandankumar.fedorapeople.org/rocky9_latest_neutron_whitebox.qcow2 We move to infra link once it is available.